source/includes/_resource.videos.md

```diff
@@ -581,57 +644,299 @@ $ curl -X PUT "https://api.vhx.tv/videos/1" \
       </td>
       <td>A description for the video.</p></td>
     </tr>
-    <tr class="is-internal text-2 border-bottom border--light-gray">
+    <tr class="text-2 border-bottom border--light-gray">
       <td>
-        <strong class="is-block text--navy">metadata</strong>
-        <span class="text--transparent text-3">object of key value pairs</span>
+        <strong class="is-block text--navy">short_description</strong>
+        <span class="text--transparent text-3">optional string, default is null</span>
+      </td>
+      <td>A shorter description for the video.</p></td>
+    </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">thumbnail_url</strong>
+        <span class="text--transparent text-3">optional string, default is null</span>
       </td>
       <td>
-      A set of key/value pairs that you can attach to a video object. It can be useful for
-      storing additional information about the video in a structured format.
-      <br><br>
-      Metadata keys must be strings. There are a few reserved keys that
-      are auto-generated, which cannot be updated. These reserved keys are:
-      <code>advertising_keywords</code>, <code>series_name</code>,
-      <code>season_name</code>, <code>season_number</code>,
-      <code>episode_number</code>, and <code>movie_name</code>
-      <br><br>
-      Metadata values can be strings, integers, arrays, or images. An image metadata value must
-      must be a url of an image, hosted on vhx, prefixed with the text <code>image_url:</code>.
+        A publicly accessible image URL. If you prefer, you can upload images directly to a video in the Vimeo OTT Publisher Admin.
       </td>
     </tr>
-    <tr class="is-internal text-2 border-bottom border--light-gray">
+    <tr class="text-2 border-bottom border--light-gray">
+      <td>
+        <strong class="is-block text--navy">url</strong>
+        <span class="text--transparent text-3">optional string, default is video title</span>
+      </td>
+      <td>
+        The final part of the url of the video, ie the <code>/video_title</code> where the video can be found on your Vimeo OTT site.
+      </td>
```

Comment:
might be worth including a full url here so something like:

> ie the `video_title` in `https://yoursite.com/video_title`

Reply:
I like that a lot 👍